homeassistant/components/habitica/__init__.py

```diff
@@ -74,9 +106,47 @@
         vol.Optional(ATTR_TASK): cv.string,
     }
 )
+SERVICE_UPDATE_TASK_SCHEMA = vol.Schema(
```

Comment:
Just wondering, wouldn't it be nice to have a `services.py` to do all the service stuff in so you can keep the init cleaner?

Reply:
Yeah, I had the same thought 😂